src/UI/TransactionsToastList/index.tsx

```diff
@@ -111,7 +232,24 @@ function TransactionsToastList({
     mapPendingSignedTransactions();
   }, [pendingTransactionsToRender]);
 
-  return <div className={generatedClasses.wrapper}>{mappedToastsList}</div>;
-}
+  useEffect(() => {
+    if (!Array.isArray(customToastsFromStore)) {
+      return;
+    }
+
+    setToastsIds((toastsIds: any[]) =>
+      customToastsFromStore.reduce(removeToastDuplicates, toastsIds)
```

Comment:
I think it's better to move the result of the reduce to a constant and use that in the function call